automation_tools/__init__.py

```diff
@@ -1032,7 +1027,9 @@ def setup_foreman_discovery(sat_version):
         run('wget -nv -O- {0} | tar x --overwrite -C /var/lib/tftpboot/boot'.format(image_url))
     else:
         # Since 6.3, installer should install all required packages except FDI
-        run('rpm -q {0}'.format(' '.join(packages)))
+        if sat_version in ('6.5', '6.6'):
+            # Check BZ 1738199 for final solution (versionlock prevents satellite rpms to install)
+            run('foreman-maintain packages unlock')
```

Comment:
Another question is Should we lock it again?

Reply:
I think we should, but I pretty sure, we do some rpm installation in tests :(, virtwho comes to mind

Reply:
it will lock next installer run which happen afterwards.
Anyway, PR solves it still better than now when we unlock permanently